ui/perfherder/alerts/AlertHeader.jsx

```diff
@@ -47,7 +47,12 @@ const AlertHeader = ({
         </Col>
         <Col className="p-0" xs="auto">
           <UncontrolledDropdown tag="span">
-            <DropdownToggle className="btn-xs ml-2" color="secondary" caret>
+            <DropdownToggle
+              className="btn-xs ml-2"
+              color="secondary"
+              caret
+              data-testid="push-dropdown"
+            >
               {alertSummary.revision.slice(0, 12)}
```

Comment:
I would add this line into a constant, since you are using it again a few lines below, this way the code is cleaner.